packages/react-log-viewer/src/LogViewer/Styles/log-viewer.css

```diff
@@ -0,0 +1,148 @@
+.pf-c-log-viewer {
```

Comment:
Is this local `Styles` folder still necessary? It should be pulling styling from the updated core now.